assimilation_code/modules/assimilation/filter_mod.dopplerfold.f90

```diff
@@ -228,10 +234,12 @@ module filter_mod
                       obs_sequence_out_name = "obs_seq.final",  &
                       adv_ens_command       = './advance_model.csh'
 
-! Inflation namelist entries follow, first entry for prior, second for posterior
-! inf_flavor is 0:none, 1:obs space, 2: varying state space, 3: fixed state_space,
-! 4: rtps (relax to prior spread), 5: enhanced state space
-integer  :: inf_flavor(2)                  = 0
+! The inflation flavor parameters are defined in adaptive_inflate_mod.
+! It is an open question as to whether or not we should 'use' the 
```

Comment:
i don't think that's an open question - to be consistent it's a good idea.  this comment could go.  it was an open question when they were simply called PRIOR and POSTERIOR.

Reply:
OK - I'll remove the comment.